app/components/post_draft/post_input/post_input.tsx

```diff
@@ -319,6 +319,7 @@ export default function PostInput({
             onChangeText={handleTextChange}
             onFocus={onFocus}
             onPaste={onPaste}
+            selection={{start: cursorPosition, end: cursorPosition}}
```

Comment:
Not sure what this change is for.

Reply:
Its for changing cursor position

Reply:
> @larkox I've updated code based on your comments also I found some bugs along the way such wrong cursor position on autocomplete for both @ mention and emoji. So I fixed those too in this PR. I'm attaching the screenshot before the fix to have better context. ![emoji](https://user-images.githubusercontent.com/43959510/265125009-71200646-d849-4784-a58a-ab3d8970f44b.jpg) ![at_mention](https://user-images.githubusercontent.com/43959510/265125016-f0f69212-5144-4172-9252-82a85901dbd9.jpg)

If we select an option in autocomplete when the cursor is between two text, the position of cursor not correct.

Reply:
I wonder if this will mess with selections in general, since (if I am reading right) if I select some text, this will overwrite that selection, and leave it unselected. Am I right?

Reply:
Yes right, I'll fix it